Classes/Controller/MatchTicker.php

```diff
@@ -466,7 +500,7 @@ protected function createFormArray($match) {
 	/**
 	 * Erstellt eine neue Spielaktion mit den Daten aus dem Request
 	 */
-	private function insertNotes($data) {

```

Comment:
Is public, weil ich die Methode überschreiben musste. Protected hätte es sicher auch getan?